src/lib.rs

```diff
@@ -528,6 +540,7 @@ impl Default for VadConfig {
             positive_speech_threshold: 0.5,
             negative_speech_threshold: 0.35,
             pre_speech_pad: Duration::from_millis(600),
+            post_speech_pad: Duration::from_millis(600),
```

Comment:
I would make this 0 by default, we can experiment with it in future to see if it improves engine performance if we notice some premature cutting.